### ISSUE TITLE
Adding label to issue when an impediment is registered

### ISSUE THREAD

**Issue by emilianodellacasa** [zerocracy/farm#1890]
I think that could be an easy and useful feature: when an user asks to wait on a issue, as soon as the bot registers the impediment, it should also add a label to the issue so that it can be easy to identify visually those issues.

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) please, pay attention to this issue

**Comment by 0crat** [comment]
@emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa) this project will fix the problem faster if you donate a few dollars to it; just [click here](https://www.0crat.com/contrib/C3NDPUA8L) and pay via Stripe, it's very fast, convenient and appreciated; thanks a lot!

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) not enough funds available in the project, can't set budget of job [#1890](https://github.com/zerocracy/farm/issues/1890), see [§21](http://www.zerocracy.com/policy.html#21); @g4s8[/z](https://www.0crat.com/u/g4s8) will get no money on completion; in order to fix that, add funds to the project and assign the job again

**Comment by 0crat** [comment]
Job `gh:zerocracy/farm#1890` already assigned to @g4s8, can't assign to @g4s8

**Comment by 0crat** [comment]
Manual assignment of issues is discouraged, see [§19](http://www.zerocracy.com/policy.html#19): -5 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by 0crat** [comment]
Bug was reported, see [§29](http://www.zerocracy.com/policy.html#29): +15 point(s) just awarded to @emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa)

**Comment by g4s8** [comment]
@0crat wait and add label

**Comment by 0crat** [comment]
> @0crat wait and add label ([here](https://github.com/zerocracy/farm/issues/1890#issuecomment-463133319))

@g4s8 The impediment for [#1890](https://github.com/zerocracy/farm/issues/1890) was registered successfully by @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by 0crat** [comment]
> @0crat wait and add label ([here](https://github.com/zerocracy/farm/issues/1890#issuecomment-463133319))

@g4s8 Job [#1890](https://github.com/zerocracy/farm/issues/1890) is not in the agenda of @g4s8[/z](https://www.0crat.com/u/g4s8), can't set impediment

**Comment by g4s8** [comment]
@emilianodellacasa it seems to be fixed, see https://github.com/zerocracy/farm/issues/1890#event-2136213465

**Comment by emilianodellacasa** [comment]
@g4s8 Thanks

**Comment by 0crat** [comment]
Job [#1890](https://github.com/zerocracy/farm/issues/1890) is not in the agenda of @g4s8[/z](https://www.0crat.com/u/g4s8), can't retrieve data and time of add

**Comment by 0crat** [comment]
Job was finished in 17 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
The job [#1890](https://github.com/zerocracy/farm/issues/1890) is now out of scope

**Comment by 0crat** [comment]
Order was finished: +35 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)
